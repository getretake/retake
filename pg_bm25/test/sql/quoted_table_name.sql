CREATE TABLE "Activity" (name TEXT, age INTEGER);
INSERT INTO "Activity" (name, age) VALUES ('Alice', 29);
INSERT INTO "Activity" (name, age) VALUES ('Bob', 34);
INSERT INTO "Activity" (name, age) VALUES ('Charlie', 45);
INSERT INTO "Activity" (name, age) VALUES ('Diana', 27);
INSERT INTO "Activity" (name, age) VALUES ('Fiona', 38);
INSERT INTO "Activity" (name, age) VALUES ('George', 41);
INSERT INTO "Activity" (name, age) VALUES ('Hannah', 22);
INSERT INTO "Activity" (name, age) VALUES ('Ivan', 30);
INSERT INTO "Activity" (name, age) VALUES ('Julia', 25);
CREATE INDEX ON "Activity" USING bm25(("Activity".*)) WITH (text_fields='{"name": {}}');
SELECT * FROM "Activity" WHERE "Activity" @@@ 'alice';

