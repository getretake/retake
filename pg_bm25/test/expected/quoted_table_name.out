CREATE TABLE "Activity" (key SERIAL, name TEXT, age INTEGER);
INSERT INTO "Activity" (name, age) VALUES ('Alice', 29);
INSERT INTO "Activity" (name, age) VALUES ('Bob', 34);
INSERT INTO "Activity" (name, age) VALUES ('Charlie', 45);
INSERT INTO "Activity" (name, age) VALUES ('Diana', 27);
INSERT INTO "Activity" (name, age) VALUES ('Fiona', 38);
INSERT INTO "Activity" (name, age) VALUES ('George', 41);
INSERT INTO "Activity" (name, age) VALUES ('Hannah', 22);
INSERT INTO "Activity" (name, age) VALUES ('Ivan', 30);
INSERT INTO "Activity" (name, age) VALUES ('Julia', 25);
CREATE INDEX ON "Activity" USING bm25(("Activity".*)) WITH (key_field='key', text_fields='{"name": {}}');
SELECT * FROM "Activity" WHERE "Activity" @@@ 'name:alice';
 key | name  | age 
-----+-------+-----
   1 | Alice |  29
(1 row)

